test/conformance/kuttl/mutate/clusterpolicy/standard/existing/target-preconditions/policy.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: update-targets
+spec:
+  background: false
+  rules:
+    - name: update-targets
+      match:
+        resources:
+          kinds:
+            - ConfigMap
+      preconditions:
+        all:
+          - key: "{{ request.object.name }}"
+            operator: Equals
+            value: trigger
+      mutate:
+        targets:
+          - apiVersion: v1
+            kind: ConfigMap
+            namespace: "{{ request.object.namespace }}"
```

Comment:
`request.object.metadata.namespace`?

Reply:
damn, was troubleshooting why it didn't work 🤦